config/SuSAv2MECPXSec.xml

```diff
@@ -7,18 +7,20 @@ Configuration for the SuSAv2-MEC cross-section model
 
 Configurable Parameters:
 .....................................................................................................
-Name                  Type     Optional   Comment                                    Default
+Name                    Type     Optional   Comment                                    Default
 .....................................................................................................
-MEC-XSecScale         double   Yes        XSec Scaling factor                        1.
-CKM-Vud               double   No         Vud element of CKM matrix                  CommonParam[CKM]
-FermiMomentumTable    string   No         Table of Fermi momentum (kF) constants     CommonParam[FermiGas]
+MEC-XSecScale           double   Yes        XSec Scaling factor                        1.
+CKM-Vud                 double   No         Vud element of CKM matrix                  CommonParam[CKM]
+QEL-CC-Qvalue2p2hShift  double   yes        Qvalue variation                 0.
```

Comment:
Add in the comments that the shift can be positive or negative. Give also an order of magnitude for reasonable changes.